integration-authorization/src/main/java/org/symphonyoss/integration/authorization/oauth/v1/OAuth1Exception.java

```diff
@@ -25,12 +25,23 @@
  */
 public class OAuth1Exception extends AuthorizationException {
 
+  private int code;
```

Comment:
I don't think we should use HTTP codes here. There's the RemoteApiException for that.